### PR TITLE
login feature

### PR DESCRIPTION
**Pull request by ppingchn**
- [x] login feature
- [x] register feature

### CODE REVIEW

File: login.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
 console.log('login');
 console.log('log out');
+console.log('register');
```
Comment (Saowaluckpruitt) [inline login.js]: cool

Comment (kanm4s) [inline login.js:3]: test comment

Comment (snexcolly123) [inline login.js:3]: kak

File: login.js [hunk 2]
```diff
@@ -1,2 +1,3 @@
 console.log('login');
 console.log('log out');
```
Comment (Saowaluckpruitt) [inline login.js:2]: grete

File: login.js (hunk 1 above)
Comment (snexcolly123) [inline login.js:3]:
```suggestion
console.log('dek hey');
```

File: login.js (hunk 2 above)
Comment (kanm4s) [inline login.js:2]:
```suggestion
console.log('kuy');
```

File: login.js [hunk 3]
```diff
@@ -1,2 +1,3 @@
 console.log('login');
```
Comment (Saowaluckpruitt) [inline login.js:1]:
```suggestion
console.log('login let me in');
```

**Review by snexcolly123 — APPROVED** [review]

**Review by kanm4s — APPROVED** [review]

**Review by Saowaluckpruitt — APPROVED** [review]

**Pull request merged**